src/main/scala/cromwell/binding/command/ParameterCommandPart.scala

```diff
@@ -38,6 +38,7 @@ case class ParameterCommandPart(attributes: Map[String, String], expression: Wdl
     }
 
     value match {
+      case b: WdlBoolean if attributes.contains("true") && attributes.contains("false") => if (b.value) attributes.get("true").head else attributes.get("false").head
```

Comment:
Can you add a negative expectation test showing what happens if one forgets true or false?